src/wp-includes/script-loader.php

```diff
@@ -2454,6 +2454,27 @@ function wp_enqueue_global_styles() {
 	wp_add_global_styles_for_blocks();
 }
 
+/**
+ * Enqueues the global styles custom css defined via theme.json.
+ *
+ * @since 6.2.0
+ */
+function wp_enqueue_global_styles_custom_css() {
+	if ( ! wp_is_block_theme() ) {
+		return;
+	}
+
+	// Don't enqueue Customizer's custom CSS separately.
+	remove_action( 'wp_head', 'wp_custom_css_cb', 101 );
```

Comment:
This has me a bit concerned: It means that, if a site is using the global styles custom CSS, Customizer custom CSS no longer works, correct?

If so, is that clear to the user? I agree that in a perfect world a site should only use one or the other, but unless it is made very clear to the end user that using the global styles custom CSS is going to override the Customizer custom CSS or unless there is some sort of migration, I would advise we don't remove this action callback.

Reply:
`$custom_css  = wp_get_custom_css();` on line 2470 gets any customizer CSS and appends it just before the new global styles custom CSS. The idea behind this is to guarantee the new global styles custom CSS has a higher specificity in load order at least. 

The decision to do this came from [some confusion in user testing](https://github.com/WordPress/gutenberg/issues/30142#issuecomment-1372950974) where users were not sure why CSS added in global styles were not overriding existing CSS added in the customizer

Reply:
Ahh, thanks for clarifying @glendaviesnz, I had missed that. All good then!